packages/relay/src/index.ts

```diff
@@ -34,6 +35,7 @@ import { TopicOnlyDecoder } from "./topic_only_message.js";
 const log = debug("waku:relay");
 
 export type Observer<T extends IDecodedMessage> = {
+  pubsubTopic: PubSubTopic;
```

Comment:
This info is already present on `IDecoder`, no? Maybe worth adding a getter to the prototype instead?